<?xml version="1.0" ?>
<launch>
  <arg name="robot" default="panda" doc="choose your robot. Possible values: [panda, fr3]"/>
  <arg name="arm_id" default="$(arg robot)" />

  <rosparam param="use_sim_time">true</rosparam>
  <rosparam command="load" file="$(find fliqc_controller_ros)/config/controller_config.yaml" subst_value="true" />
  <rosparam command="load" file="$(find fliqc_controller_ros)/config/rosbag_record_profile.yaml" subst_value="true" />

  <node pkg="debug_and_profile_helper" type="record_as_rosbag.py" name="record_as_rosbag" output="screen" required="true">
    <param name="allow_overwrite" value="true"/>
  </node>

  <include file="$(find fliqc_controller_ros)/launch/gazebo_sim.launch" pass_all_args="true">
    <arg name="controller" value="fliqc_joint_velocity_no_env_node_profile" />
  </include>
  
  <node pkg="rviz" type="rviz" output="screen" name="rviz" args="-d $(find fliqc_controller_ros)/launch/sim_fliqc_joint_velocity_no_env_node_debug.rviz -f $(arg arm_id)_link0 --splash-screen $(find franka_visualization)/splash.png"/>
</launch>